src/common/models/Projection.ts

```diff
@@ -435,7 +432,9 @@ export class Projection {
     // TODO(michael): I don't like hardcoding "County" into the name. We should
     // probably delete this code and get the location name from somewhere other
     // than the API (or improve the API value).
```

Comment:
Can you remove this TODO?  It's outdated (at some point the API started including 'County' in the names...)